.github/workflows/round-explorer.yml

```diff
@@ -0,0 +1,29 @@
+name: package/round-explorer - Build -> Test
+on:
+  push:
+  pull_request:
+    branches:
+      - main
+jobs:
+  build-and-deploy:
+    concurrency: ci-${{ github.ref }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+
+      - name: Install Dependencies
+        run: |
+          yarn install
+          cd packages/round-explorer
+          yarn install
+
+      - name: Build Round Explorer
+        run: |
+          yarn re-build
```

Comment:
at first i thought "re-build" meant "rebuild" as in "build again" 🤦‍♀️ 
did we settle on "round explorer" vs "grant explorer"? some things use GE (milestone name, notion docs, etc) but some use RE 🤔

Reply:
LOL i think it's round-explorer :P that's what it intially set at the time of repo creation 
There is this other vocabulary where grants are called projects so then it could be project explorer 😂 

We should bring this up on standup but atm the entire repo and it's dependencies all are called round explorer

Reply:
Ooo i see nate has pointed out that GE is the actual name. Renaming everything :P